placement/objects/allocation_candidate.py

```diff
@@ -132,7 +132,8 @@ def _get_by_requests(cls, context, groups, rqparams, nested_aware=True):
         candidates = {}
         for suffix, group in groups.items():
             rg_ctx = res_ctx.RequestGroupSearchContext(
-                context, group, rw_ctx.has_trees, sharing, suffix)
+                context, group, rw_ctx.has_trees, sharing, suffix,
+                rqparams.ignore_consumers)
```

Comment:
Did you think about introducing this per `RequestGroup` instead having it global only in the `RequestWideParams`? So that one can specify different ignore_consumers per group.
If yes, I'd like to have an insight about the decision.

Reply:
When ignoring a consumer there can be 2 expectations:
1) ignore all allocations of that consumer for this request. this would stretch all groups.
2) ignore it only for a group
I opted for the first one, because no matter what group of resource-providers our consumer is in, when we want to resize, we want to ignore all groups, as all might get changes. Additionally, the version of Nova we're currently using, doesn't know about request groups, yet, afaik.